plugins/TelescopeControl/src/INDI/TelescopeClientINDI.cpp

```diff
@@ -94,8 +94,26 @@ void TelescopeClientINDI::telescopeGoto(const Vec3d &positionJ2000, StelObjectP
 void TelescopeClientINDI::telescopeSync(const Vec3d &positionJ2000, StelObjectP selectObject)
 {
 	Q_UNUSED(selectObject)
-	Q_UNUSED(positionJ2000)
-	return;
+	const StelCore* core = StelApp::getInstance().getCore();
+	Vec3d posRectJNow = core->j2000ToEquinoxEqu(positionJ2000, StelCore::RefractionOff);
+	Vec3d posJ2000;
+	StelUtils::rectToSphe(&posJ2000[0], &posJ2000[1], posRectJNow);
+
+	if (posJ2000[0] < 0.0)
+		posJ2000[0] += 2*M_PI;
+
+	double longitudeRad = posJ2000[0] * 12.0 / M_PI;
+	double latitudeRad = posJ2000[1] * 180.0 / M_PI;
+
+
+	INDIConnection::Coordinates positionJNow = mConnection.position();
+	positionJNow.RA = longitudeRad;
+	positionJNow.DEC = latitudeRad;
+
+	// unpark telescope before slewing!
+	// TODO: Add commands and buttons for park/unpark telescope for all telescopes
+	mConnection.unParkTelescope();
```

Comment:
I do not think this is the place to call *this* particular command.
It's fine to 'make sure' the mount is unparked before giving it a sync command in another way, but we should not unpark at every sync.

Reply:
The mount will not unpark here if she is unparked already. But possible we should add 2 buttons (Park, Unpark) for all telescopes to allow better management of mounts.